python/desc/sims/GCRCatSimInterface/InstanceCatalogWriter.py

```diff
@@ -195,6 +282,15 @@ def write_catalog(self, obsHistID, out_dir='.', fov=2):
 
             gal_cat.write_catalog(os.path.join(out_dir, gal_name), chunk_size=100000,
                                   write_header=False)
+        
+        # SN instance catalogs
+        for i, snpop in enumerate(snpopcsvs):
+            phosimcatalog = snphosimcat(snpop, tableName=names[i],
+                                        obs_metadata=obs_md, objectIDtype=i+42)
+
+            snOutFile = names[i] +'_cat_{}.txt'.format(obsHistID)  
```

Comment:
In the future, I would like to consider creating the supernova catalog names in their own list that can be appended to `[star_name, gal_name]`.  We can then use that list of supernova catalog names here.  That will reduce the likelihood of lines 291 and 209 drifting out of sync.